debian/pt-os-web-portal-wpa-gui-vnc.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=pi-topOS wpa_gui over VNC
+Documentation=https://knowledgebase.pi-top.com/knowledge
+
+[Service]
+Type=simple
+RemainAfterExit=yes
+Restart=no
+ExecStart=/usr/bin/pt-web-vnc start --display-id 99 --run "wpa_gui" --height 800 --width 1000 --with-window-manager --background-colour white
```

Comment:
there is a (.1%) chance of this conflicting with a further-link vnc request with the id 99. maybe we should restrict further to a smaller range or something?